extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/ConflictsWithScrutinizer.java

```diff
@@ -0,0 +1,58 @@
+package org.openrefine.wikidata.qa.scrutinizers;
+
+import org.openrefine.wikidata.qa.QAWarning;
+import org.openrefine.wikidata.updates.ItemUpdate;
+import org.wikidata.wdtk.datamodel.interfaces.PropertyIdValue;
+import org.wikidata.wdtk.datamodel.interfaces.Statement;
+import org.wikidata.wdtk.datamodel.interfaces.Value;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ConflictsWithScrutinizer extends EditScrutinizer {
+
+    public static final String type = "having-conflicts-with-statements";
+
+    @Override
+    public void scrutinize(ItemUpdate update) {
+        Map<PropertyIdValue, Value> propertyIdValueValueMap = new HashMap<>();
+        for (Statement statement : update.getAddedStatements()){
+            PropertyIdValue pid = statement.getClaim().getMainSnak().getPropertyId();
+            Value value = statement.getClaim().getMainSnak().getValue();
```

Comment:
In the case of NoValueSnak & SomeValueSnak, it will make a call to the method implemented in SnakImpl which will return null. So, I guess it will not create any issues. If it is an issue, please let me know, I'll use some other way to implement the functionality.

Reply:
That is fine then! It would be worth adding a test case with a `NoValueSnak` to demonstrate that, perhaps?